src/lcmap/commons/tile.clj

```diff
@@ -1,12 +1,69 @@
-(ns lcmap.commons.tile
-  (:require [clojure.tools.logging :as log]))
+(ns lcmap.commons.tile)
+
+(defn difference
+  "Calculate difference between a point and 'prior' point on an interval.
+
+  The value of this function can be used to answer the question,
+  what do I subtract from a point to find the point of the nearest
+  tile that contains it?
+
+  Geospatial raster data geometry can be somewhat counter-inuitive
+  because coordinates and pixel geometry are expressed with both
+  positive and negative values.
+
+  Along the x-axis, pixel size (and thus the interval) is a positive
+  number (e.g. 30 * 100). Along the y-axis though, the pixel size and
+  interval is a _negative_ value. Even though this may seem peculiar,
+  using a negative value helps us avoid special cases for finding the
+  nearest tile-point."
+  [point interval]
```

Comment:
Use of term 'point' implies x & y together.  Can we either adjust terminology or define what a point is in comments?